src/stories/components/UserGreeting/UserGreeting.test.js

```diff
@@ -0,0 +1,46 @@
+import { findByText, render, screen } from '@testing-library/react';
```

Comment:
I think findByText is the unused variable that Fernando mentioned

Reply:
Good catch! Thank you